roles/edpm_pre_adoption_validation/molecule/adoption-negative/molecule.yml

```diff
@@ -0,0 +1,32 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: podman
+platforms:
+- command: /sbin/init
+  dockerfile: ../../../../molecule/common/Containerfile.j2
+  image: ${EDPM_ANSIBLE_MOLECULE_IMAGE:-"ubi9/ubi-init"}
+  name: edpm-0
+  privileged: true
+  registry:
+    url: ${EDPM_ANSIBLE_MOLECULE_REGISTRY:-"registry.access.redhat.com"}
+  ulimits:
+  - host
+provisioner:
+  name: ansible
+  inventory:
+    hosts:
+      all:
+        hosts:
+          edpm-0:
+            canonical_hostname: edpm-0.localdomain
```

Comment:
the test files have oldname.localdomain and here we are setting it to edpm-0.localdomain

so this is what forces the failure.

Reply:
yes